.travis.yml

```diff
@@ -30,11 +28,8 @@ script:
   - qiita_env make --no-load-ontologies
   - if [ ${TEST_ADD_STUDIES} == "True" ]; then test_data_studies/commands.sh ; fi
   - if [ ${TEST_ADD_STUDIES} == "False" ]; then qiita_test_install ; fi
-  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests --with-doctest --with-coverage ; fi
+  - if [ ${TEST_ADD_STUDIES} == "False" ]; then nosetests -v --with-doctest --with-coverage ; fi
```

Comment:
Saw your last commit message, any idea why? Note that IMOO the -v makes things less clear ... but I guess we can get used to the new output format